src/zope/testrunner/eggsupport.py

```diff
@@ -27,29 +27,30 @@ def skipLayers(suite, _result=None):
     return _result
 
 class SkipLayers(ScanningLoader):
-    """ Load only unit tests (those which have no layer associated with them).
+    """Load only unit tests (those which have no layer associated with
+    them).
 
-    o Running the tests using 'setup.py test' cannot, by default, drive the
-      full testrunner, with its support for layers (in functional tests).
-      This loader allows the command to work, by running only those tests
-      which don't need the layer support.
+    o Running the tests using 'setup.py test' cannot, by default,
+    drive the full testrunner, with its support for layers (in
+    functional tests).  This loader allows the command to work, by
+    running only those tests which don't need the layer support.
```

Comment:
Does ReST support lowercase `o` as a bullet marker?  I'd no idea.  In any case, if this is a bulleted list, then 2nd and all other lines should be indented an extra two characters like they were before you reformatted.

I suggest switching the bullet market to `*` or `-` in make it clearer.

Reply:
Good idea, thanks.